app/src/main/java/org/dpppt/android/app/main/MainFragment.java

```diff
@@ -67,6 +71,18 @@ private void setupHeader(View view) {
 		headerView = view.findViewById(R.id.main_header_container);
 		tracingViewModel.getAppStateLiveData()
 				.observe(getViewLifecycleOwner(), appState -> headerView.setState(appState));
+
+		tracingViewModel.getNetworkEnableLiveData()
```

Comment:
Adds new liveData to intercept changes of the network connection by change the UI according to this input